src/ui/atoms/checkbox/index.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import styled, { StyledComponent } from 'styled-components';
+
+interface CheckboxProps {
+  id: string;
+  isChecked: boolean;
+  onChange(event: React.SyntheticEvent): void;
+}
+
+export const Checkbox: React.FC<CheckboxProps & { className: string }> = ({
```

Comment:
```suggestion
const Checkbox: React.FC<CheckboxProps & { className: string }> = ({
```

Думаю, что пользователю не нужен доступ к базовому компоненту без стилей.

Иначе пользователь получит доступ к `checkbox.Checkbox` и может быть удивлен, почему это верстка расползлась

Reply:
Удалишь `export` ?

Reply:
@sergeysova сорян, забыла удалить старый Checkbox, компонент лежит в molecules/toggle